stock_move_manual_lot/views/stock_move_line.xml

```diff
@@ -0,0 +1,22 @@
+<odoo>
+    <record id="view_move_line_form" model="ir.ui.view">
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_move_line_form" />
+        <field name="arch" type="xml">
+            <form position="inside">
+                <field name="use_manual_lot_selection" invisible="True" />
+            </form>
+            <field name="lot_id" position="before">
+                <field
+                    name="manual_lot_id"
+                    groups="stock.group_production_lot"
+                    context="{'default_product_id': product_id, 'active_picking_id': picking_id}"
+                    attrs="{'invisible': ['|', '|', ('use_manual_lot_selection', '=', False), ('lot_id', '!=', False), ('lot_name', '!=', False)]}"
```

Comment:
This way, `manual_lot_id` is invisible when either
* the` functionality is disabled for the picking type, or
* lot_id is set, or
* a name for a new lot is set

Isn't it the case that lot_id is already set when the move line is reserved with just any lot (which is our basic use case)? In that case, we will want to see this field I think.

Reply:
yes, I think now it's more sensible

Reply:
Thanks! You may squash now.